### ISSUE TITLE
Labeled.bind() ignore the given format

### ISSUE THREAD

**Issue by slimaku** [edvin/tornadofx#210]
Hello, 

When using the `Labeled.bind()` method for a number with a `Format`, the given format doesn't have any effect.

Here is a test that reproduce my problem :

```kotlin
@Test
fun testBindingFormat() {
	JFXPanel() // Initialize the JFX toolkit

	val property = SimpleDoubleProperty(Math.PI)

	val label = Label()
	label.bind(property, format = DecimalFormat("#0.0"))

	Assert.assertEquals("3.1", label.text)
}
```

The problem is in the function `bindStringProperty` which do :
```kotlin
// ...
val effectiveConverter = converter ?: getDefaultConverter<T>()
if (readonly) {
	val toStringConverter = Callable {
		if (converter != null)
			converter.toString(property.value)
		else if (format != null)
			format.format(property.value)
		else property.value?.toString()
	}
	val stringBinding = Bindings.createStringBinding(toStringConverter, property)
	stringProperty.bind(stringBinding)
} else {
	if (effectiveConverter != null) {
		stringProperty.bindBidirectional(property, effectiveConverter)
	} else if (format != null) {
		stringProperty.bindBidirectional(property, format)
	} else {
		throw IllegalArgumentException("Cannot convert from ${T::class} to String without an explicit converter or format")
	}
}
// ...
```

So, with a Double Property, there will always be a default converter and the format will be ignored.

**Comment by edvin** [comment]
Thanks for reporting this. The issue has been fixed in 1.5.8-SNAPSHOT now :) I added your test to the test suite, but changed it slightly so it forces a specific Locale. The test failed with my default locale (Norwegian, since we use `,` as the decimal separator). The modified test looks like this:

```kotlin
    @Test
    fun testBindingFormat() {
        val property = SimpleDoubleProperty(Math.PI)

        val label = Label()
        label.bind(property, format = DecimalFormat("#0.0", DecimalFormatSymbols.getInstance(Locale.ENGLISH)))

        Assert.assertEquals("3.1", label.text)
    }
```

**Comment by slimaku** [comment]
Great, thank you.
